lib/honeycomb/configuration.rb

```diff
@@ -12,19 +12,25 @@ class Configuration
                   :debug
 
     attr_writer :service_name, :client, :host_name
+    attr_reader :error_backtrace_limit
 
     def initialize
       @write_key = ENV["HONEYCOMB_WRITEKEY"]
       @dataset = ENV["HONEYCOMB_DATASET"]
       @service_name = ENV["HONEYCOMB_SERVICE"]
       @debug = ENV.key?("HONEYCOMB_DEBUG")
+      @error_backtrace_limit = 0
       @client = nil
     end
 
     def service_name
       @service_name || dataset
     end
 
+    def error_backtrace_limit=(val)
+      @error_backtrace_limit = Integer(val)
```

Comment:
Get a `TypeError` at config time if you set a non-numerical value here rather than waiting until later to either not get your values or something.